src/utils.js

```diff
@@ -4,8 +4,9 @@ const news = axios.create({
   baseURL: "https://news-project-g3uj.onrender.com/api",
 });
 
-const getArticles = async () => {
-  const { data } = await news.get("/articles");
+const getArticles = async (queryString) => {
```

Comment:
Could we rename this file to api, it is a util but api would be more specific to what the functions purpose are.